fastchat/train/train.py

```diff
@@ -277,7 +277,8 @@ def train():
         padding_side="right",
         use_fast=False,
     )
-    tokenizer.pad_token = tokenizer.unk_token
+    if tokenizer.pad_token and tokenizer.pad_token != tokenizer.unk_token:
```

Comment:
This is wrong for llama.
`tokenizer.pad_token` is `None` for llama

Reply:
Thanks for pointing this bug, Have fixed this and tested with llama successfully.